Gemfile.lock

```diff
@@ -106,27 +106,208 @@ GEM
       rspec-expectations (~> 2.13.0)
       rspec-mocks (~> 2.13.0)
     slop (3.6.0)
-    sprockets (2.12.1)
+    sprockets (2.12.3)
       hike (~> 1.2)
       multi_json (~> 1.0)
       rack (~> 1.0)
       tilt (~> 1.1, != 1.3.0)
-    sprockets-rails (2.1.3)
+    sprockets-rails (2.2.2)
       actionpack (>= 3.0)
       activesupport (>= 3.0)
-      sprockets (~> 2.8)
+      sprockets (>= 2.8, < 4.0)
     sqlite3 (1.3.7)
     thor (0.19.1)
     thread_safe (0.3.4)
     tilt (1.4.1)
     treetop (1.4.15)
       polyglot
+      polyglot
```

Comment:
FWIW I had that same weird thing happen. I actually had dozens of lines of "polyglot."  Did the same "fix" - manually reverted the change, after which it didn't happen again after rerunning `bundle`.  Might be a bundler bug.